/energe_detect_node:
  ros__parameters:
    ThresholdGray: 125
    ThresholdColor: 170